nntrainer/tensor/blas_interface.cpp

```diff
@@ -326,8 +326,8 @@ static void sgemm_FP16(CBLAS_ORDER order, CBLAS_TRANSPOSE TransA,
                        const unsigned int ldc) {
 
 #if (defined USE__FP16 && USE_NEON)
-  nntrainer::neon::hgemm(A, B, C, M, N, K, alpha, beta, TransA == CblasTrans,
-                         TransB == CblasTrans);
+  nntrainer::neon::custom_hgemm(A, B, C, M, N, K, alpha, beta,
```

Comment:
Is there any reason for ```custom_```?

Reply:
Because there can be many half-precision gemm solutions in the future, and `blas_interface` should be just an interface to choose from them.
On top of that, I just wanted to split `hgemm` function totally from `blas_interface`

Reply:
Then, I think hgemm is more proper function name for the users and we can manage internal functions name according to the various hgemm version.

Reply:
Sure. will be refactored properly in the later patch